PhysicsTools/PatAlgos/python/slimming/MicroEventContent_cff.py

```diff
@@ -76,26 +76,37 @@
         'keep CTPPSLocalTrackLites_ctppsLocalTrackLiteProducer_*_*'
     )
 )
-MicroEventContentMC = cms.PSet(
-    outputCommands = cms.untracked.vstring(MicroEventContent.outputCommands)
-)
-MicroEventContentMC.outputCommands += [
-        #'keep *_slimmedGenJets*_*_*',
-        'keep *_slimmedGenJets_*_*',
-        'keep *_slimmedGenJetsAK8_*_*',
+
+MicroEventContentGEN = cms.PSet(
+    outputCommands = cms.untracked.vstring(
         'keep patPackedGenParticles_packedGenParticles_*_*',
         'keep recoGenParticles_prunedGenParticles_*_*',
         'keep LHEEventProduct_*_*_*',
-        'keep PileupSummaryInfos_slimmedAddPileupInfo_*_*',
         'keep GenFilterInfo_*_*_*',
         'keep GenLumiInfoHeader_generator_*_*',
         'keep GenLumiInfoProduct_*_*_*',
         'keep GenEventInfoProduct_generator_*_*',
+        'keep *_slimmedGenJets_*_*',
+        'keep *_slimmedGenJetsAK8_*_*',
+        'keep *_genMetTrue_*_*',
+        'keep *_genMetCalo_*_*',
+        'keep *_genMetCaloAndNonPrompt_*_*',
```

Comment:
@gpetruc 
based on https://github.com/cms-sw/cmssw/pull/19853#issuecomment-322242838
I was expecting only genMetTrue. 
This change adds more.
Looking at AODSIM content, we have only genMetTrue and genMetCalo in the events.
Is this desired in miniAOD?